spec/main.tex

```diff
@@ -896,8 +896,13 @@ \subsubsection{Non-Threshold (i.e., $n$-out-of-$n$) Key Generation}
           and set $X_i = x_i \cdot G$.
     \item Sample $\rid_i \gets \{0,1\}^\kappa$.
     \item Compute $(A_i; \tau_i) = \commit{sch}()$.
+    \item {\bf (HD-wallets.)} 
+    \begin{itemize}
+      \item If HD-wallets support enabled, sample local chain code contribution $c_i \gets \{0,1\}^{256}$ (32-bytes string)
+      \item Otherwise, set $c_i = ``"$ (to empty string)
+    \end{itemize}
```

Comment:
Might be better to say `set c_i = ⊥` or `nil`? Or even omit mentioning it's set to anything in the otherwise case. I'm concerned about the interpretation of hashing and sending of empty string, but I'm not sure which one reads clearer.

Reply:
Changed to $\bot$